src/shell/shell.cpp

```diff
@@ -1241,15 +1243,15 @@ void SHELL_Init() {
 	MSG_Add("SHELL_CMD_PATH_HELP_LONG",
 	        "Usage:\n"
 	        "  [color=light-green]path[reset]\n"
-	        "  [color=light-green]path[reset] [color=light-cyan][[drive:]path[;...][reset]\n"
+	        "  [color=light-green]path[reset] [color=light-cyan][[DRIVE:]PATH[;...][reset]\n"
 	        "\n"
 	        "Where:\n"
-	        "  [color=light-cyan][[drive:]path[;...][reset] is a path containing a drive and directory.\n"
+	        "  [color=light-cyan][[DRIVE:]PATH[;...][reset] is a path containing a drive and directory.\n"
```

Comment:
In all honestly, I'd get rid of these "is" words everywhere. My ideal version looks like this:

```
  [DRIVE:]PATH[;...]  path containing a drive and directory
```

So two spaces at least between the option name and the description for clear separation, and all lowercase, imperative mood, and no period at the end.

If there's a long block of text of multiple fully-formed sentences, we can use normal sentences (starts with uppercase, period at the end, etc.)

Reply:
I'd remove the "is" words too...
Perhaps "Where:" could be changed to "Parameters:" or "Arguments:"?
I think as it is now it reads fine ("Where ARGUMENT is..." or "Where /p does..."), but with the changes you propose renaming "Where" would be much better.

Reply:
Yeah, I agree "arguments" would be better, yes. I think we should break away from this concept that this should read as a whole sentence starting from "Where" -- there's not much benefit of that.